python_modules/dagster/dagster/core/execution/plan/execute_plan.py

```diff
@@ -200,8 +200,7 @@ def dagster_event_sequence_for_step(
         (6) Framework failure:
```

Comment:
interesting that case (5) is raised as a FRAMEWORK_ERROR despite (6) being described here as a framework failure...

Reply:
ya its a mismatch between describing the cause of the error versus describing the error it self 

ie a `DagsterError` comes from the framework but is caused by a problematic usage 

feel free to more liberally update these comments, they are quite old